property_tests/tests/tuple/test/test.ts

```diff
@@ -94,12 +95,19 @@ function generateBody(
     `;
 }
 
+function replacer(key: any, value: any) {
```

Comment:
What is this replacer for? is there a JSON.stringify somewhere? We should use the replacer and reviver I've created now, perhaps we should open an issue for this or comment it in here

Reply:
Remove this